Ginger/GingerCoreNET/ActionsLib/ActionsCobversion/ActionConversionUtils.cs

```diff
@@ -0,0 +1,265 @@
+﻿using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Common.UIElement;
+using Amdocs.Ginger.Repository;
+using GingerCore;
+using GingerCore.Actions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+namespace Amdocs.Ginger.CoreNET
+{
+    /// <summary>
+    /// This class is used to add methods for action conversion helpers
+    /// </summary>
+    public class ActionConversionUtils
+    {
+        public string ActUIElementElementLocateByField
```

Comment:
please fix this class FOLDER name, there is typo in "ActionsCo-b-version"

Reply:
Done